lib/pavilion/plugins/results/table.py

```diff
@@ -41,13 +41,27 @@ def get_config_items(self):
                           "nested dictionaries by columns. Default False. "
                           "Only set if `has_header` is True. "
                           "Otherwise, Pavilion will ignore."
+            ),
+            yc.StrElem(
+                'title', 
+                help_text="Partial or full title of table."
```

Comment:
What is this used for? 

(I'm sure that will be apparent through the code, but it needs to be apparent in the help_text too).